### DISCUSSION TITLE
What is abci++?

### DISCUSSION THREAD

**Discussion by pcpLiu** [tendermint/spec#379] [Q&A] [answered]
Hi,

We are recently learning tendermint spec for our chain dev. On the tendermint core doc, there's a section [abci++](https://docs.tendermint.com/v0.35/spec/abci++/v4.html#). But there's no context in it. 

Is it an on-going draft of next-gen spec?

Thanks!

**Comment by josef-widder** [comment, marked as answer]
Hi! 

@sergio-mena is currently leading our efforts in extending the ABCI++ spec. The work is happening in PR https://github.com/tendermint/spec/pull/366. There are minor TODOs in the document but otherwise it is pretty stable. It would be great if you could have a look and let us know what kind of additional information would be valuable for you so that we can incorporate your suggestions. Within the PR, the file spec/abci++/README.md is the entry point.

Thanks!

**Reply by pcpLiu** [reply]
Thanks!
